tests/fake_device/raop.py

```diff
@@ -53,6 +100,9 @@ class FakeRaopState:
     def __init__(self):
         self.metadata = SimpleNamespace(title=None, artist=None, album=None)
         self.audio_packets: Dict[int, bytes] = {}  # seqo -> raw audio
+        self.password = None
+        self.nonce = None
+        self.realm = "raop"
```

Comment:
Maybe just put as a constant at the top?

Reply:
In theory realm can have different values. According to the unofficial Airplay documentation the Apple TV allows `AirPlay` and `raop`. Therefore I decided against a constant. I'm not quite sure, when which value is used.